x-pack/plugins/alerting/server/application/rule/schemas/flapping_schema.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+
+export const flappingSchema = schema.object({
+  lookBackWindow: schema.number(),
```

Comment:
I don't think we should validate these since these are simply describing what the schema looks like. We're validating the I/O types so it should be ok

Reply:
Ok! Thanks!